terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/reservetour/ReserveTourHelperTest.java

```diff
@@ -23,7 +23,6 @@
 import static org.mockito.Mockito.when;
 
 import org.dozer.DozerBeanMapper;
-import org.dozer.Mapper;
```

Comment:
- [ ] It seems better to change here to use `Mapper` interface.
https://github.com/terasolunaorg/terasoluna-tourreservation/pull/617/commits/f92c442c46ae7be75fe2249837fa011237b252a2#diff-4999d7ce579a5186b218aa8598886e9dR69

Please see also https://github.com/terasolunaorg/terasoluna-tourreservation/blob/5.5.x/terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/reservetour/ReserveTourHelperTest.java#L70

Reply:
2ea2f311aa24ec0d19428db8872abe8680b40020
Modified to DozerBeanMapper -> Mapper.
In addition, added code with comments to the mapping file example (ManageCustomerControllerTest.java).

In the same section of 5.5.x, the same mapping files sample as 5.4.x are described
([ManageCustomerControllerTest.java#L65-L67(5.5.x)](https://github.com/terasolunaorg/terasoluna-tourreservation/blob/5.5.x/terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managecustomer/ManageCustomerControllerTest.java#L65-L67)). 
This 5.5.x code sample seems to need to be fixed.
Reference: [ReserveServiceImplTest.java#L89-L92(5.5.x)](https://github.com/terasolunaorg/terasoluna-tourreservation/blob/5.5.x/terasoluna-tourreservation-domain/src/test/java/org/terasoluna/tourreservation/domain/service/reserve/ReserveServiceImplTest.java#L89-L92).

Reply:
The following is helpful .

https://github.com/DozerMapper/dozer/blob/master/docs/asciidoc/migration/v6-to-v61.adoc#3-dozerbeanmapper-usage-is-deprecated